targets/TARGET_Ambiq_Micro/TARGET_Apollo3/sdk/mcu/apollo3/hal/am_hal_uart.c

```diff
@@ -420,7 +420,7 @@ am_hal_uart_configure(void *pHandle, const am_hal_uart_config_t *psConfig)
     UARTn(ui32Module)->LCRH = (psConfig->ui32DataBits   |
                                psConfig->ui32Parity     |
                                psConfig->ui32StopBits   |
-                               AM_HAL_UART_FIFO_ENABLE);
+                               AM_HAL_UART_FIFO_DISABLE);
```

Comment:
does / should this HAL change go through our AmbiqSuiteSDK mirror branch?

is there really no way to change this setting via HAL calls w/ proper configuration?